dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/components/IntegerComponent.kt

```diff
@@ -0,0 +1,68 @@
+package org.dataland.frameworktoolbox.intermediate.components
+
+import org.apache.commons.text.StringEscapeUtils
+import org.dataland.frameworktoolbox.intermediate.FieldNodeParent
+import org.dataland.frameworktoolbox.specific.datamodel.TypeReference
+import org.dataland.frameworktoolbox.specific.datamodel.elements.DataClassBuilder
+import org.dataland.frameworktoolbox.specific.fixturegenerator.elements.FixtureSectionBuilder
+import org.dataland.frameworktoolbox.specific.uploadconfig.elements.UploadCategoryBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.SectionConfigBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.getTypescriptFieldAccessor
+import org.dataland.frameworktoolbox.specific.viewconfig.functional.FrameworkDisplayValueLambda
+
+/**
+ * A IntegerComponent represents a numeric value from the integer domain
+ */
+open class IntegerComponent(
+    identifier: String,
+    parent: FieldNodeParent,
+) : ComponentBase(identifier, parent) {
+    open var constantUnitSuffix: String? = null
```

Comment:
`DecimalComponent` has `(minimum|maximum)Value` fields. Why not this one? If not provided via template. They definitely should be.

Reply:
Ok but I just checked the code base, and it turns out that `minimumValue` is only used in a test, and `maximumValue` is not used at all.
So we could also remove these class variables without losing anything.

The other option would be to include it in the `IntegerComponent` too, but then we should also actually do smth with it, like e.g. passing a validator for this into the UploadConfig, or generating the fixtures accordingly.
Furthermore we would need the possibility to pass min and max values via the excel template.
However this sounds like it's not done in 10min.   So should we rather do it in some other ticket?

Reply:
Yup. Let's not do this here. But maybe remove it in `DecimalComponent` then. I'll leave this up to you. I'll just let this conversation open till the end of the day for you to have the chance to read this answer.

Reply:
I think removing it makes sense. It is dead code.

So pls leave this open until I have removed it.
I will notify you then.

Reply:
Removed it and also adjusted the test which used "minimumValue". See the commit

Reply:
👍